sdk/test/Services/S3/UnitTests/Custom/ResponseMapperTests.cs

```diff
@@ -268,6 +268,96 @@ public void MapAbortMultipartUploadsCommand_ConstructAbortMultipartUploadRequest
             Assert.AreEqual(RequestPayer.Requester, result.RequestPayer, "RequestPayer should be set");
         }
 
+        [TestMethod]
+        [TestCategory("S3")]
+        public void Validate_DownloadRequest_ConvertToGetObjectRequest_MapsAllProperties()
+        {
+            var tuDownloadRequest = new TransferUtilityDownloadRequest
+            {
+                BucketName = "test-bucket",
+                Key = "test-key",
+                VersionId = "v-1",
+                ModifiedSinceDate = Amazon.Util.AWSSDKUtils.CorrectedUtcNow.AddDays(-2),
+                UnmodifiedSinceDate = Amazon.Util.AWSSDKUtils.CorrectedUtcNow.AddDays(-1),
+                ServerSideEncryptionCustomerMethod = ServerSideEncryptionCustomerMethod.AES256,
+                ServerSideEncryptionCustomerProvidedKey = "base64-key",
+                ServerSideEncryptionCustomerProvidedKeyMD5 = "md5",
+                ExpectedBucketOwner = "expected-owner",
+                IfMatch = "etag-match",
+                IfNoneMatch = "etag-notmatch",
+                ResponseHeaderOverrides = new ResponseHeaderOverrides { CacheControl = "no-cache" },
+                RequestPayer = RequestPayer.Requester,
+                ChecksumMode = ChecksumMode.ENABLED
+            };
+
+            var cmd = new DownloadCommand(null, tuDownloadRequest);
```

Comment:
is there a reason we cant use ValidateMappingTransferUtilityAndSdkRequests like in MapUploadRequest_AbortMultipartRequest_AllMappedProperties_WorkCorrectly?